xchange-blockchain/src/main/java/org/knowm/xchange/blockchain/BlockchainExchange.java

```diff
@@ -1,32 +1,56 @@
 package org.knowm.xchange.blockchain;
 
 import org.knowm.xchange.BaseExchange;
-import org.knowm.xchange.Exchange;
 import org.knowm.xchange.ExchangeSpecification;
+import org.knowm.xchange.blockchain.service.BlockchainAccountService;
+import org.knowm.xchange.client.ExchangeRestProxyBuilder;
+import org.knowm.xchange.client.ResilienceRegistries;
 import si.mazi.rescu.SynchronizedValueFactory;
 
-/** @author Tim Molter */
-public class BlockchainExchange extends BaseExchange implements Exchange {
+import javax.ws.rs.HeaderParam;
+
+import static org.knowm.xchange.blockchain.BlockchainConstants.*;
+
+/** @author scuevas*/
+public class BlockchainExchange extends BaseExchange {
```

Comment:
I think we will also need to override service methods (marketDataService, tradingService), to indicate we are not supporting them yet, as you do with getNonceFactory

Reply:
you mean something like this?

```
this.accountService = new BlockchainAccountService(this, this.blockchain, this.getResilienceRegistries());
this.marketDataService = (MarketDataService) new UnsupportedOperationException("Not implemented yet");
this.tradeService = (TradeService) new UnsupportedOperationException("Not implemented yet");
```